tests/ext/get_memory_limit_default.phpt

```diff
@@ -1,11 +1,11 @@
 --TEST--
 Test get_memory_limit() returning correct values for default
 --INI--
-memory_limit=100
+memory_limit=10000000
```

Comment:
I understand an increase, but is it expected to be this large of an increase?

Reply:
the minimum memory_limit is 2M. so using 10M to make the 80% totally obvious.

Reply:
Can you leave a comment about this? I know there wasn't one before, but I don't know why this test would be 80%, or anything about 2M min memory_limit ^_^

Reply:
Sure, added :-)